src/engraving/libmscore/beam.cpp

```diff
@@ -372,18 +372,22 @@ void Beam::layout1()
     //
     // determine beam stem direction
     //
+    ChordRest* firstNote = _elements.front();
```

Comment:
You should check whether the list is empty or not

Reply:
done!